src/fmeval/constants.py

```diff
@@ -11,29 +13,48 @@
 SAGEMAKER_SERVICE_ENDPOINT_URL = "SAGEMAKER_SERVICE_ENDPOINT_URL"
 SAGEMAKER_RUNTIME_ENDPOINT_URL = "SAGEMAKER_RUNTIME_ENDPOINT_URL"
 
-# Constants for column names in loaded ray datasets
-MODEL_INPUT_COLUMN_NAME = "model_input"
-MODEL_OUTPUT_COLUMN_NAME = "model_output"
-MODEL_LOG_PROBABILITY_COLUMN_NAME = "model_log_probability"
-TARGET_OUTPUT_COLUMN_NAME = "target_output"
-CATEGORY_COLUMN_NAME = "category"
-SENT_MORE_INPUT_COLUMN_NAME = "sent_more_input"
-SENT_LESS_INPUT_COLUMN_NAME = "sent_less_input"
-SENT_MORE_PROMPT_COLUMN_NAME = "sent_more_prompt"
-SENT_LESS_PROMPT_COLUMN_NAME = "sent_less_prompt"
-SENT_LESS_LOG_PROB_COLUMN_NAME = "sent_less_log_prob"
-SENT_MORE_LOG_PROB_COLUMN_NAME = "sent_more_log_prob"
-
-COLUMN_NAMES = {
-    MODEL_INPUT_COLUMN_NAME,
-    MODEL_OUTPUT_COLUMN_NAME,
-    TARGET_OUTPUT_COLUMN_NAME,
-    CATEGORY_COLUMN_NAME,
-    SENT_MORE_INPUT_COLUMN_NAME,
-    SENT_LESS_INPUT_COLUMN_NAME,
-    SENT_MORE_LOG_PROB_COLUMN_NAME,
-    SENT_LESS_LOG_PROB_COLUMN_NAME,
-}
+
+class ColumnNames(Enum):
+    """
+    This enum represents the names of columns that appear
+    in the finalized Ray Dataset produced by an EvalAlgorithm.
+    These are the only columns whose data gets written to output
+    records by util.save_dataset. Other algorithm-specific columns
+    that get produced as intermediate results (for example,
+    CLASSIFIED_MODEL_OUTPUT_COLUMN_NAME in ClassificationAccuracy)
+    are not included here, and thus won't get saved by save_dataset.
+    """
+
+    MODEL_INPUT_COLUMN_NAME = "model_input"
+    PROMPT_COLUMN_NAME = "prompt"
+    MODEL_OUTPUT_COLUMN_NAME = "model_output"
+    MODEL_LOG_PROBABILITY_COLUMN_NAME = "model_log_probability"
+    TARGET_OUTPUT_COLUMN_NAME = "target_output"
+    CATEGORY_COLUMN_NAME = "category"
+    SENT_MORE_INPUT_COLUMN_NAME = "sent_more_input"
+    SENT_LESS_INPUT_COLUMN_NAME = "sent_less_input"
+    SENT_MORE_PROMPT_COLUMN_NAME = "sent_more_prompt"
+    SENT_LESS_PROMPT_COLUMN_NAME = "sent_less_prompt"
+    SENT_MORE_LOG_PROB_COLUMN_NAME = "sent_more_log_prob"
+    SENT_LESS_LOG_PROB_COLUMN_NAME = "sent_less_log_prob"
+
+
+COLUMN_NAMES = [e.value for e in ColumnNames]
+
+# These constants are included so that eval algorithm code doesn't need
```

Comment:
Subjective:
I think we should remove these, it's fine being verbose in the code: `ColumnName.MODEL_INPUT_COLUMN_NAME.value`. It keeps the code readable, ensures in readers mind that `ColumnNames` enum is the source of truth.

Reply:
Makes sense; will update